src/components/CustomInput/CustomInput.tsx

```diff
@@ -91,37 +91,37 @@ function InputSearch(props: TInputProps, ref: TInputComponentRef) {
   return (
     <div className={classNames(containerClass)}>
       <div className={scss.inputWrapper}>
-        <label htmlFor={props.name} className={classNames(labelClass)}>
+        <label className={classNames(labelClass)} htmlFor={props.name}>
           {props.placeholder}
         </label>
         <div className={scss.iconLeft}>
-          <img src={leftIcon} />
+          <Icon icon="mag" />
         </div>
         <input
-          required
-          readOnly={props.readOnly}
-          type={props.type}
+          className={classNames(inputClass)}
           id={props.name}
           name={props.name}
-          className={classNames(inputClass)}
-          placeholder={props.placeholder}
           onBlur={(e) => onInputBlur(e)}
+          onChange={handleValue}
           onFocus={onInputFocus}
+          placeholder={props.placeholder}
+          readOnly={props.readOnly}
+          required
+          type={props.type}
```

Comment:
você pode usar o `spread` pra essas props que são padrões do input pra facilitar a vida, assim quem usar esse componente vai ter acesso as props que você criou + as props nativas de um input, incluindo as que você não passou aqui
```suggestion
         {...props}
```